apps/language_server/test/server_test.exs

```diff
@@ -1136,7 +1136,7 @@ defmodule ElixirLS.LanguageServer.ServerTest do
 
   test "returns code lenses for runnable tests", %{server: server} do
     in_fixture(__DIR__, "test_code_lens", fn ->
-      file_path = "test/fixture_test.exs"
```

Comment:
I wonder if this is a good approach. `mix test` skips `.ex` files by default.

Reply:
I agree it feels a little janky and I'm not sure if it's gonna hold when we add support for ExUnit's `test_paths` and `test_pattern`.

The best solution in my opinion would be to delete the `elixir_ls_utils/test/tmp` folder between test runs, but I don't know if this has any ramifications I don't know about.

Reply:
So far I have not seen any negative consequences of deleting `elixir_ls_utils/test/tmp`. The other solution I have in mind is moving the temp folder to e.g. `elixir_ls_utils/.tmp`

Reply:
I hadn't thought of moving the folder, but that's a pretty good idea! I can't think of any reason why it would cause problems and my limited testing (running `mix test` a bunch of times in all apps and umbrella root) it seems to work fine.